numba/testing/main.py

```diff
@@ -807,3 +808,62 @@ def run(self, test):
         # This will call self._run_inner() on the created result object,
         # and print out the detailed test results at the end.
         return super(ParallelTestRunner, self).run(self._run_inner)
+
+
+
+
+import xmlrunner
```

Comment:
Let's make this dependency (unittest-xml-reporting) optional. 
And, put it at the top of the file. (sorry, this is probably from my hacky commit )